ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/dropdown/v1/dropdown/dropdown.html

```diff
@@ -36,26 +36,51 @@
-        <div data-sly-list.enumName="${dropdown.enumNames}" data-sly-unwrap>
-            <div data-sly-test="${enumName != null && dropdown.enums[enumNameList.index] != null}" data-sly-unwrap>
-                <option value="${dropdown.enums[enumNameList.index].toString}"
-                        class="cmp-adaptiveform-dropdown__option"
-                        data-sly-attribute.selected="${dropdown.enums[enumNameList.index] in dropdown.default ? 'selected' : ''}">${enumName}</option>
+    <div data-sly-test="${!dropdown.isEnableSuggestions}" data-sly-unwrap>
+        <select class="cmp-adaptiveform-dropdown__widget"
+                id="${widgetId}"
+                title="${dropdown.tooltipVisible ? '' : dropdown.tooltipText}"
+                name="${dropdown.name}"
+                data-cmp-data-layer="${dropdown.data.json}"
+                disabled="${!dropdown.enabled || dropdown.readOnly}"
+                required="${dropdown.required}"
+                data-sly-attribute.multiple="${dropdown.isMultiSelect ? 'multiple' : ''}"
+                aria-readonly="${dropdown.readOnly ? 'true' : ''}">
+            <div data-sly-test.isPlaceHolderSet="${dropdown.placeHolder != null}" data-sly-unwrap></div>
+            <option data-sly-test="${!isPlaceHolderSet}" disabled selected></option>
+            <option data-sly-test="${isPlaceHolderSet}" value="" disabled selected>${dropdown.placeHolder}</option>
+            <div data-sly-list.enumName="${dropdown.enumNames}" data-sly-unwrap>
+                <div data-sly-test="${enumName != null && dropdown.enums[enumNameList.index] != null}" data-sly-unwrap>
+                    <option value="${dropdown.enums[enumNameList.index].toString}"
+                            class="cmp-adaptiveform-dropdown__option"
+                            data-sly-attribute.selected="${dropdown.enums[enumNameList.index] in dropdown.default ? 'selected' : ''}">${enumName}</option>
+                </div>
             </div>
-        </div>
-    </select>
+        </select>
+    </div>
+    <div data-sly-test="${dropdown.isEnableSuggestions}" data-sly-unwrap>
+        <datalist id="${widgetId}">
+            <div data-sly-list.enumName="${dropdown.enumNames}" data-sly-unwrap>
+                <div data-sly-test="${enumName != null && dropdown.enums[enumNameList.index] != null}"
+                     data-sly-unwrap>
+                    <option value="${dropdown.enums[enumNameList.index].toString}"
+                            class="cmp-adaptiveform-dropdown__option"
+                            data-sly-attribute.selected="${dropdown.enums[enumNameList.index] in dropdown.default ? 'selected' : ''}">
+                        ${enumName}
+                    </option>
+                </div>
+            </div>
+        </datalist>
+        <input list="${widgetId}" class="cmp-adaptiveform-dropdown__widget"
+               id="${widgetId}"
```

Comment:
right, thanks rishi! I will fix this as well.

Reply:
Hi, rishi! I have updated the classname for data-list to maintain the uniqueness with id. Select and Input in data-list have same id since at one time only one of the component will be available in DOM tree. 
But inside the data-list component, i have changed the id of data-list keeping the id for input as it is.